ember_debug/vendor/startup-wrapper.js

```diff
@@ -38,21 +38,24 @@ if (typeof env !== 'undefined') {
       Ember.EmberInspectorDebugger.Adapter = requireModule('ember-debug/adapters/' + adapter)['default'];
 
       onApplicationStart(function appStarted(app) {
+        var isFirstBoot = !('__inspector__booted' in app);
         app.__inspector__booted = true;
         Ember.EmberInspectorDebugger.set('application', app);
         Ember.EmberInspectorDebugger.start(true);
-        // Watch for app reset
-        app.reopen({
-          reset: function() {
-            this.__inspector__booted = false;
-            this._super.apply(this, arguments);
-          },
-          willDestroy: function() {
-            Ember.EmberInspectorDebugger.destroyContainer();
-            Ember.EmberInspectorDebugger.set('application', null);
-            this._super.apply(this, arguments);
-          }
-        });
+        if (isFirstBoot) {
+          // Watch for app reset/destroy
+          app.reopen({
```

Comment:
its worth noting, if this mixin wasn't anonymous duplicate reopens of the same mixin wouldn't result in the duplicate being applied.